### PR TITLE
Hotellist

### PR DESCRIPTION
**Pull request by surajnarayan1**
@say2prabhash

### CODE REVIEW

File: UI/SearchPage/room_script.js [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+    for(var i=0;i<roomItinerary.Itinerary.Rooms.length;i++)
+        {
+            typeOfRooms.push({
+               roomType:roomItinerary.Itinerary.Rooms[i].RoomName,
+                roomDescription:roomItinerary.Itinerary.Rooms[i].RoomDescription,
+                bedType:roomItinerary.Itinerary.Rooms[i].BedType,
+                roomFare:roomItinerary.Itinerary.Rooms[i].DisplayRoomRate.TotalFare.Amount,
+            });
+        }
+    var template = $('#hotel-item');
```
Comment (say2prabhash) [inline UI/SearchPage/room_script.js]: change it to room-item because it's for room.

File: UI/SearchPage/room_script.js [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
-    contentType:"application/json",
-  cache: false,
-    crossDomain:"true",
-    dataType:"json",
-  success: successFunction
-});
-
+	  var html = compiledTemplate(hotelList);
 
+	  $('#hotelList-container').html(html);
```
Comment (say2prabhash) [inline UI/SearchPage/room_script.js]: change it to room-list container.

File: RoomDetail.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+
+<body>
+
+  <h1>Hotel List page</h1>
+
+  <div id="hotelList-container">
+  </div>
+
+  <script id="hotel-item" type="text/x-handlebars-template">
```
Comment (tavisca-pmohanty) [inline RoomDetail.html]: change from hotel-item to room-item

**Review by tavisca-pmohanty — CHANGES_REQUESTED** [review]

File: RoomDetail.html [hunk 4]
```diff
@@ -1,29 +1,44 @@
...
-    </div>
+<button type="submit" id="room-button" value="{{name}}">BookNow</button>
+
  {{/each}}
   </script>
 
   <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
   <script src="http://cdnjs.cloudflare.com/ajax/libs/handlebars.js/2.0.0/handlebars.min.js"></script>
-  <script src="main.js" type=""></script>
+  <script src="RoomDetail.js"></script>
```
Comment (tavisca-pmohanty) [inline RoomDetail.html]: file name should be in lower case

File: UI/SearchPage/listing-page.js [hunk 5]
```diff
@@ -77,21 +77,13 @@
 			         function getSuccess(data)
 			          {
 			             
-<<<<<<< HEAD
+
 			            	var roomItineraries=data;
             				sessionStorage.setItem('RoomListing',JSON.stringify(roomItineraries));
 
-             				window.location="roomlist.html";
+             				window.location="RoomDetail.html";
```
Comment (tavisca-pmohanty) [inline UI/SearchPage/listing-page.js]: file name should be in lower case.

**Review by tavisca-pmohanty — CHANGES_REQUESTED** [review]

**Review by tavisca-pmohanty — APPROVED** [review]

**Pull request merged**